README.md

```diff
@@ -51,4 +58,4 @@ Start the development server on `http://localhost:3000`:
 npm run dev
 ```
 
-Now you can open a browser and navigate to `http://localhost:3000/research-software-support`
\ No newline at end of file
+Now you can open a browser and navigate to `http://localhost:3000/research-software-support`
```

Comment:
Hmm, I'm not sure why this was not there in the first place. Thank you for adding!

Reply:
It was there! Probably a changed line ending or something...